.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "Appium/tests/helpers/mocks"]
 	path = Appium/tests/helpers/mocks
 	url = https://github.com/jasonkarns/bats-mock
+[submodule "Appium/tests/helpers/bats"]
```

Comment:
@andrcuns Thanks for the PR and it makes sense to move things out of container. I was thinking is there any better way of doing it instead of using bats a sub module, can't we use their node package?